src/js/pages/organizer/Branding.jsx

```diff
@@ -1,9 +1,122 @@
+import AuthService from "util/AuthService";
+import BrandStore from "stores/BrandStore";
+import dispatcher from "dispatcher";
 import React from "react";
 
+var moment = require('moment');
+const url = "https://dev.calligre.com";
+const $ = require("jquery");
+
 export default class Branding extends React.Component {
+  constructor(props) {
+    super(props);
 
-  render() {
-  	return (<div> Branding </div>);
+    this.state = {
+      name: '',
+      organization: '',
+      starttime: '',
+      endtime: '',
+      location: '',
+      other: '',
+      color_primary: '#000000',
+      color_secondary: '#000000',
+      logo: '',
+      logo_square: '',
+      icon: '',
+      facebook: '',
+      twitter: '',
+    };
+
+    BrandStore.on("receivedBranding", this.setBranding);
+    BrandStore.on("savedBranding", this.brandingSaved)
+    BrandStore.getBranding();
+  }
+
+  setBranding = () => {
+    let branding = BrandStore.branding;
+    
+    Object.keys(branding).forEach(function(key) {
+      if (key == 'starttime' || key == 'endtime') {
+        branding[key] = moment.unix(branding[key]).format('YYYY-MM-DD[T]HH:mm');
+      }
+    });
+    this.setState(branding);
   }
 
-}
\ No newline at end of file
+  handleChange = (event) => {
+    this.setState({[event.target.name]:event.target.value});
+  }
+
+  handleSave = (event) => {
+    const data = {
+      name: this.state.name,
+      organization: this.state.organization,
+      starttime: moment(this.state.starttime).unix(),
+      endtime: moment(this.state.endtime).unix(),
+      location: this.state.location,
+      other: this.state.other,
+      color_primary: this.state.color_primary,
+      color_secondary: this.state.color_secondary,
+      logo: this.state.logo,
+      logo_square: this.state.logo_square,
+      icon: this.state.icon,
+      facebook: this.state.facebook,
+      twitter: this.state.twitter,
+    };
+
+    BrandStore.saveBranding(data);
+  }
+
+  brandingSaved = (event) => {
+    //TODO: Figure out a better way of alerting the user that saving succeeded
+    console.log("Branding saved successfully.");
+  }
+
+  render() {
+    return (
+      <div>
+        <table width="100%" height="100%">
+          <tbody>
+            <tr>
+              <td width="50%">
```

Comment:
This is, I think, necessary. It's a table with two elements, each having half the full width.

I could remove it, but I think it'll have an undefined, maybe half, maybe not, look to it.